### PR TITLE
Add docs for mnist tensorflow

### PR DESCRIPTION
**Pull request by sub-mod**

### CODE REVIEW

**Comment by mattf** [comment]
@zmhassan @willb @elmiko ptal

**Comment by mattf** [comment]
this is a great addition. it should be followed w/ a screenshot or two to show what the tutorial produces.

File: _applications/tensorflow_mnist.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+instructions:
+
+
+1. Ensure that you are connected to an OpenShift project
+	
+	oc new-project test
+
+2. Create the template for mnist web application.
+
+   oc create -f https://raw.githubusercontent.com/sub-mod/mnist-app/master/template.json
```
Comment (willb) [inline _applications/tensorflow_mnist.adoc]: Subin, this is great.  My only quibble is this:  should we host `mnist-app` under the organization account?

Comment (willb) [reply _applications/tensorflow_mnist.adoc]: ok, fine with me to take it as an action to fix in the near future

File: _applications/tensorflow_mnist.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+== Introduction
+
+MNIST digit identification application provides an example of how to use s2i build to deploy a tensorflow prediction endpoint on Openshift.
+Input data is taken from  http://yann.lecun.com/exdb/mnist/[MNIST site].
+
+
+[[architecture]]
+== Architecture
+
+The system is composed of:
```
Comment (elmiko) [inline _applications/tensorflow_mnist.adoc]: are these all in a single pod, multiple pods, etc.

i think we should say something like:

"The system is composed of 3 pods:"

but make it fit the deploy, if it's all 1 pod that's fine, i just think we should state that.

File: _applications/tensorflow_mnist.adoc [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+	oc new-app --template=mnistapp --param=APPLICATION_NAME=mnist-app \
+		--param=PREDICTION_SERVICE1=tf-reg \
+		--param=PREDICTION_SERVICE2=tf-cnn
+
+
+[[usage]]
+== Usage
+
+The `mnistapp` template creates a route on Openshift, so you should be able to access the web application by clicking on the link provided in the UI.
```
Comment (elmiko) [inline _applications/tensorflow_mnist.adoc]: s/so you should be able to access/you should now be able to access/

File: _applications/tensorflow_mnist.adoc [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+You can now begin drawing digits on the canvas and queries will be sent to
+the two prediction endpoints.The response from the prediction endpoints are displayed on the UI.
```
Comment (elmiko) [inline _applications/tensorflow_mnist.adoc]: s/displayed on the UI./displayed on the UI and should look similar to the following image:/

**Comment by tmckayus** [comment]
agree with earlier comments, other than that lgtm!

**Comment by zmhassan** [comment]
I have another example app that uses inception pretrained model. I'm not using the same approach as you did with mnist. I'll look at how I can use this s2i image to deploy that application.

**Comment by zmhassan** [comment]
Wonder if you can have a base-tf image with https://github.com/zmhassan/radanalytics-image-recognition/blob/master/Dockerfile#L11-L97

File: _applications/tensorflow_mnist.adoc [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+:page-labels: [Tensorflow, S2I]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: This demo shows how to use source-to-image Tensorflow Serving build to deploy a tensorflow serving prediction endpoint on Openshift. The s2i build provides a GRPC microservice endpoint for web applications to send queries to be evaluated against the tensorflow model.
+:page-project_links: ["https://github.com/radanalyticsio/tensorflow-serving-s2i"]
+
+[[introduction]]
+== Introduction
+
+The MNIST digit identification application provides an example of how to use https://github.com/openshift/source-to-image[source-to-image] to build to deploy a tensorflow prediction endpoint on Openshift.
```
Comment (elmiko) [inline _applications/tensorflow_mnist.adoc]: s/to build to deploy/to build and deploy/

**Comment by elmiko** [comment]
lgtm, thanks Subin!

i think we should add something to link back to the "get started" page as the other tutorials do, and also a little more detail about what is inside this app. like what is the user creating and what does it do. but i'm ok adding these as an extra patch.

**Pull request merged**